torch_xla/experimental/tpu.py

```diff
@@ -172,7 +172,7 @@ def configure_topology(local_rank: int,
   tpu_env = get_tpu_env()
 
   accelerator_type = tpu_env[xenv.ACCELERATOR_TYPE]
-  if version() == 4:
+  if tpu_env['ACCELERATOR_TYPE'].startswith('v4') or tpu_env['ACCELERATOR_TYPE'].startswith('v5'):
```

Comment:
@will-cromar what's the right place to add a test for this one? I felt like we had similar tests somwehre.

Reply:
The relevant unit test is here: https://github.com/pytorch/xla/blob/18674daf0c44bef0162cbeddaaef7acc206ecea5/test/pjrt/test_experimental_tpu.py#L90

Can we just do `tpu.version() >= 4` here?

Reply:
tried version() >=4, failed with https://paste.googleplex.com/5365404993060864